src/my-view1.html

```diff
@@ -196,24 +282,32 @@ <h3>Select a location</h3>
 
       <section id="preview">
         <ryr-email-templatizer templatevars="{{apivars}}" htmlrenderedtemplate="{{buffer}}" markdownrenderedtemplate="{{md}}"></ryr-email-templatizer>
-        <div class="section-title bg-secondary">
-          <div class="circle">2</div>
+        <div class="section-title bg-primary">
+          <!-- <div class="circle">2</div> -->
           <h3>Preview</h3>
         </div>
-        <ryr-preview previewtitle="Preview" htmlrenderedtemplate="[[buffer]]"></ryr-preview>
+        <div class="placeholder">
+          <p>This is a placeholder for the email text.</p>
+        </div>
+        <!-- <ryr-preview previewtitle="Preview" htmlrenderedtemplate="[[buffer]]"></ryr-preview> -->
         <granite-clipboard text="[[md]]">
-          <!-- <iron-icon class="icon-medium" icon="vaadin:copy-o"></iron-icon> -->
-          <paper-icon-button icon="vaadin:copy-o" title="copy"></paper-icon-button>
+          <br/>
+          <div class="copy-icon-text">
+        <div class="circle">
+        <iron-icon class="icon-medium" icon="vaadin:copy-o"></iron-icon>
+        <!-- <paper-icon-button class="icon-big" icon="vaadin:copy-o" title="copy"></paper-icon-button> -->
```

Comment:
We should keep the `paper-icon-button` element as it provides a nice ripple effect when we click it.